libraries/bsg_manycore.cpp

```diff
@@ -461,13 +447,13 @@ static int hb_mc_manycore_format_cache_op_request_packet(hb_mc_manycore_t *mc,
 {
         int r;
 
-        if ((r = hb_mc_manycore_format_request_packet(mc, pkt, npa)) != 0)
+        if ((r = hb_mc_manycore_format_request_packet(mc, pkt, npa)) HB_MC_SUCCESS)
```

Comment:
!=

Reply:
HA. Shows I didn't even run compilation. Yes, I screwed something up in my rebase.